test/e2e/specs/published-content/forms_submission.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @group jetpack-wpcom-integration
+ */
+
+import {
+	DataHelper,
+	EditorPage,
+	TestAccount,
+	envVariables,
+	getTestAccountByFeature,
+	envToFeatureKey,
+	JetpackFormsInboxPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser, Locator } from 'playwright';
+
+const formData = {
+	name: `${ DataHelper.getRandomPhrase() }`,
+	// Making the email unique to each run allows us to filter down to one in the inbox later.
+	email: `test${ DataHelper.getTimestamp() + DataHelper.getRandomInteger( 0, 100 ) }@example.com`,
```

Comment:
**question:non-blocking** - do you want to switch to using the EmailClient.getTestEmailAddress` method I implemented in https://github.com/Automattic/wp-calypso/pull/81437?

Reply:
Eh, we can, but just in case it generates any noise, I'd rather not have that fill up mailosaur 😄